2014/01/28 09:22:22 [error] [exception.CException] exception 'CException' with message 'UsersController and its behaviors do not have a method or closure named "sendJSON".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CComponent.php:266
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php(78): CComponent->__call('sendJSON', Array)
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php(78): UsersController->sendJSON(Array)
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CInlineAction.php(49): UsersController->actionCreate()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('create')
#12 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/create')
#13 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/server/users/create
HTTP_REFERER=http://social.loc/public/
---
2014/01/28 09:30:06 [warning] [application] Failed to set unsafe attribute "wait" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (74)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:36:10 [warning] [application] Failed to set unsafe attribute "silent" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (74)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:36:48 [warning] [application] Failed to set unsafe attribute "silent" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (74)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:39:00 [warning] [application] Failed to set unsafe attribute "silent" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (49)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:41:21 [warning] [application] Failed to set unsafe attribute "silent" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (49)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:41:32 [warning] [application] Failed to set unsafe attribute "silent" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (49)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:44:33 [warning] [application] Failed to set unsafe attribute "validate" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (49)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/01/29 08:45:26 [warning] [application] Failed to set unsafe attribute "validate" of "Users".
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (49)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/02/04 08:16:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "authenticate".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('authenticate')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('authenticate')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/authentic...')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/users/authenticate
HTTP_REFERER=http://social.loc/public/
---
2014/02/05 07:23:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "logout".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('logout')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('logout')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/logout')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/users/logout
HTTP_REFERER=http://social.loc/public/
---
2014/02/05 07:47:27 [error] [php] Trying to get property of non-object (D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php:109)
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): UsersController->runAction()
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): UsersController->filterAccessControl()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): UsersController->runActionWithFilters()
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): UsersController->run()
#8 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CWebApplication->run()
REQUEST_URI=/server/users/login
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (109)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/02/10 06:19:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "id".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('id')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('id')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/id')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/users/id/98
HTTP_REFERER=http://social.loc/public/
---
2014/02/11 07:27:15 [error] [php] Trying to get property of non-object (D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php:37)
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): UsersController->runAction()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): UsersController->filterAccessControl()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#8 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): UsersController->runActionWithFilters()
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): UsersController->run()
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CWebApplication->run()
REQUEST_URI=/server/users/getuser/id/null
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (37)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/02/11 07:27:26 [error] [php] Trying to get property of non-object (D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php:37)
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): UsersController->runAction()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): UsersController->filterAccessControl()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#8 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): UsersController->runActionWithFilters()
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): UsersController->run()
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#12 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CWebApplication->run()
REQUEST_URI=/server/users/getuser/id/null
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php (37)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/02/13 07:13:51 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/13 07:20:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/13 07:57:18 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "search".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('search')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('search')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/search/va...')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/users/search/value/undefined
HTTP_REFERER=http://social.loc/public/
---
2014/02/13 08:15:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/13 08:29:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/17 08:45:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/17 08:46:24 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:24:32 [error] [exception.CException] exception 'CException' with message 'CWebApplication and its behaviors do not have a method or closure named "session".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CComponent.php:266
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php(58): CComponent->__call('session', Array)
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php(58): CWebApplication->session('role')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CInlineAction.php(49): UsersController->actionGetrole()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('getrole')
#12 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/getrole')
#13 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#14 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/server/users/getrole
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:30:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:30:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:31:04 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:31:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "news/getall".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php:286
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/getall')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/server/news/getall
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:36:28 [error] [php] Undefined variable: usersCollection (D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\NewsController.php:15)
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(286): NewsController->runAction()
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): NewsController->runActionWithFilters()
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): NewsController->run()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CWebApplication->run()
REQUEST_URI=/server/news/getall
in D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\NewsController.php (15)
in D:\EasyPHP\data\localweb\sicoal.loc\server\index.php (13)
2014/02/19 08:41:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "search".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('search')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('search')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/search/val...')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/news/search/value/n
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:41:34 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "search".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('search')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('search')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/search/val...')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/news/search/value/ne
HTTP_REFERER=http://social.loc/public/
---
2014/02/19 08:41:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "search".' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php:483
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(270): CController->missingAction('search')
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('search')
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('news/search/val...')
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/server/news/search/value/new
HTTP_REFERER=http://social.loc/public/
---
2014/03/03 08:00:58 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [2002] No connection could be made because the target machine actively refused it.
' in D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\CDbConnection.php:382
Stack trace:
#0 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\CDbConnection.php(330): CDbConnection->open()
#1 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\CDbConnection.php(308): CDbConnection->setActive(true)
#2 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CModule.php(387): CDbConnection->init()
#3 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(450): CModule->getComponent('db')
#4 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\ar\CActiveRecord.php(634): CApplication->getDb()
#5 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\ar\CActiveRecord.php(667): CActiveRecord->getDbConnection()
#6 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\db\ar\CActiveRecord.php(1458): CActiveRecord->getCommandBuilder()
#7 D:\EasyPHP\data\localweb\sicoal.loc\server\protected\controllers\UsersController.php(36): CActiveRecord->find('98')
#8 [internal function]: UsersController->actionGetuser('98')
#9 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#10 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(UsersController), Object(ReflectionMethod), Array)
#11 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(291): CFilterChain->run()
#18 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(282): CController->run('getuser')
#20 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\web\CWebApplication.php(141): CWebApplication->runController('users/getuser/i...')
#21 D:\EasyPHP\data\localweb\sicoal.loc\server\framework\base\CApplication.php(180): CWebApplication->processRequest()
#22 D:\EasyPHP\data\localweb\sicoal.loc\server\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/server/users/getuser/id/98
HTTP_REFERER=http://social.loc/public/
---
